cylc/flow/scripts/cylc_spawn.py

```diff
@@ -41,25 +38,39 @@
 
 def get_option_parser():
     parser = COP(
-        __doc__, comms=True, multitask=True,
+        __doc__, comms=True, multitask_nocycles=True,
         argdoc=[
-            ('REG', 'Suite name'),
-            ('[TASK_GLOB ...]', 'Task matching patterns')])
+            ("REG", "Suite name"),
+            ('TASK-GLOB [...]', 'Task match pattern')])
+
+    parser.add_option(
+        "--failed",
+        help="Spawn on <target-tasks>:fail.",
+        action="store_true", default=False, dest="failed")
+
+    parser.add_option(
+        "--non-failed",
+        help="Spawn on all non-failure outputs.",
+        action="store_true", default=False, dest="non_failed")
```

Comment:
Can save manually checking for mutually exclusive options like this:

```suggestion
        action="store_false", default=False, dest="failed")
```

Reply:
Trouble is these are just a part of the full solution: spawn off of `:succeed`, `:fail`, `(non-fail)`, and `:custom`.  I'll consider finishing this command off properly today...

Reply:
(spawn on specified - including custom - outputs now implemented)